### PR TITLE
Search

### PR DESCRIPTION
**Pull request by rachellehanna**
Just the beginnings of the search feature

### CODE REVIEW

File: styles/partials/_sections.scss [hunk 1]
```diff
@@ -99,6 +99,27 @@ ul li {
     }
 }
 
+//Results section
```
Comment (AlixWagner) [inline styles/partials/_sections.scss:102]: All of this green section I think is redundant to the section above
See: Line 15

File: javascript/app.js [hunk 2]
```diff
@@ -237,3 +242,23 @@ cocktailApp.revealButton.addEventListener("click", function () {
             // show recipe section
         });
 });
+
+cocktailApp.searchButton.addEventListener("click", function () {
```
Comment (AlixWagner) [inline javascript/app.js]: because this is a submit by nature we just need to add the prevent default

**Review by AlixWagner — CHANGES_REQUESTED** [review]
Awesome start - looking good!

File: styles/partials/_sections.scss (hunk 1 above)
Comment (rachellehanna) [reply styles/partials/_sections.scss:102]: oops merge conflict gone wrong!

File: javascript/app.js (hunk 2 above)
Comment (rachellehanna) [reply javascript/app.js]: added!

**Review by AlixWagner — APPROVED** [review]
Awesome! Looks great!

**Pull request merged**